ravenframework/Optimizers/BayesianOptimizer.py

```diff
@@ -235,6 +236,8 @@ def initialize(self, externalSeeding=None, solutionExport=None):
       self.raiseAnError(RuntimeError, f'GPR ROM <target> should be obective variable: {self._objectiveVar}, '
                         f'Received {self._model.supervisedContainer[0].target}')
 
+    if self._resetModel:
```

Comment:
I can't find any logic in this file that would switch the default value of `False` for `_resetModel` to `True`. Do you expect this value to change in this before this branch of code?

Reply:
This variable has been defined in  __init__ method. Currently, the default is False. I define this variable to allow us to define a XML node in the input file to control it if needed. Currently, the default behavior works fine.